proof/invariant-abstract/DetSchedSchedule_AI.thy

```diff
@@ -220,6 +220,7 @@ abbreviation valid_sched_except_blocked_2 where
 abbreviation valid_sched_except_blocked :: "'z state \<Rightarrow> bool" where
   "valid_sched_except_blocked s \<equiv> valid_sched_except_blocked_2 (ready_queues s) (etcbs_of s) (scheduler_action s) (cur_domain s) (kheap s) (cur_thread s) (idle_thread s)"
 
+\<comment> \<open>FIXME: remove\<close>
```

Comment:
for this PR?

Reply:
It's not really related to this PR so I wasn't planning to, but it violates enough of our guidelines and was easy enough that I've now done it.